web/templates/right_sidebar.hbs

```diff
@@ -18,7 +18,7 @@
                 </button>
             </div>
             <div id="buddy_list_wrapper" class="scrolling_list" data-simplebar>
-                <ul id="user_presences" class="filters" data-search-results-empty="{{t 'No matching users.' }}"></ul>
+                <ul id="narrow-user-presences" class="filters" data-search-results-empty="{{t 'No matching users.' }}"></ul>
```

Comment:
What do you think about calling this `buddy-list-users-matching-view`? This aligns more closely with the string "no matching users", and elsewhere we're using "presence" mainly to refer to just the dots, not an entire user row.

Reply:
I thought "no matching users" is specifically about "matching" the search string, not the narrow -- so I would find this name more confusing.

If we want to remove presence, I'd be okay with `buddy-list-narrow-users`

Reply:
Well that's why I put "matching view" in the name, not "matching filter".

I don't like the use of "narrow" because (1) the name is generally being phased out of the UI as a misnomer in favor of things like "messages view" or "message feed" and probably the code will follow and (2) we will likely be doing this in things like "recent conversations filtered to a stream", which does not match the historical definition of a "narrow".

Reply:
Ok sure :)